src/api.js

```diff
@@ -0,0 +1,5 @@
+const url = "https://nc-games-bib6.onrender.com/api";
+
+export const getReviews = () => {
+  return fetch(url + "/reviews").then((res) => res.json());
+};
```

Comment:
Its up to you, but you'll find it easier down the line if you use axios; especially when it comes to queries.